src/config.c

```diff
@@ -336,8 +340,37 @@ gboolean parse_config (GromitData *data)
                               goto cleanup;
                             }
                           arrowsize = scanner->value.v_float;
+                          arrowtype = GROMIT_ARROW_END;
                         }
                       else if ((intptr_t) scanner->value.v_symbol == 4)
+                        {
```

Comment:
Please use "start", "end", "double" instead of the ASCII arrows here. This makes the intent clearer.